.circleci/config.yml

```diff
@@ -197,11 +197,10 @@ workflows:
           name: "checkstyle-and-sevntu"
           image-name: *cs_img
           command: "./.ci/validation.sh checkstyle-and-sevntu"
-      # until https://github.com/spotbugs/spotbugs-maven-plugin/issues/806
-      # - validate-with-maven-script:
-      #    name: "spotbugs-and-pmd"
-      #    image-name: *cs_img
-      #    command: "./.ci/validation.sh spotbugs-and-pmd"
+      - validate-with-maven-script:
```

Comment:
Please add comment above this job `https://github.com/spotbugs/spotbugs-maven-plugin/issues/806 explains why we need execution of spotbug without any other maven plugins which can change binaries `